setup.cfg

```diff
@@ -15,7 +15,7 @@ doctest_optionflags = NORMALIZE_WHITESPACE IGNORE_EXCEPTION_DETAIL ALLOW_UNICODE
 
```

Comment:
Moving the setup.cfg from backend to root doesn't seem correct, since setup.cfg is a Python specific configuration file and backend seems to be the only directory containing a Python project here. Why do you move it?

Reply:
I think I moved it because the CI style checks were not finding the config file in the backend folder. But with for example `cd backend && flake8` it should work :+1:

Reply:
Moved back.